src/main/java/com/spotlightspace/core/attachment/service/AttachmentService.java

```diff
@@ -0,0 +1,56 @@
+package com.spotlightspace.core.attachment.service;
+
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.spotlightspace.common.entity.TableRole;
+import com.spotlightspace.core.attachment.domain.Attachment;
+import com.spotlightspace.core.attachment.repository.AttachmentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class AttachmentService {
+    private final AttachmentRepository attachmentRepository;
+
+    private final AmazonS3Client amazonS3Client;
```

Comment:
```suggestion
public class AttachmentService {

    private final AttachmentRepository attachmentRepository;

    private final AmazonS3Client amazonS3Client;
```
컨벤션을 이렇게 했던 것 같은데 확인 한번 부탁드려요🙏
인텔리제이에서 ctrl + alt + l을 누르면 코드 자동 정렬 기능을 사용할 수 있어요!